api/handler/submit-survey.js

```diff
@@ -20,6 +20,7 @@ function submitSurvey (request, reply) {
     const questionOption = database.sequelize.model('question_option');
     const surveyInstanceId = request.payload.surveyInstanceID;
     const questionInstArr = [];
+    const requestSuccess = 500;
```

Comment:
I understand that 200 would make sense but this would require an app side logic/code change.
We can ask Dr. Gary on how to proceed with the same.